test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -105,16 +105,67 @@ TEST(help_add_test, add_option)
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
 
-    // Tests the --copyright call. TODO: should be uncommented/fixed after --copyright is actually implemented.
-    // const char * argvCopyright[] = {"./copyright", "--copyright"};
-    // argument_parser copyright("copyright", 2, argvCopyright);
-    // copyright.info.long_copyright = "long copyright";
-    // testing::internal::CaptureStdout();
-    // EXPECT_THROW(copyright.parse(), parser_interruption);
-    // stdout = testing::internal::GetCapturedStdout();
-    // expected = "whatever is expected";
-    // EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
-                               // expected | ranges::view::remove_if(is_space)));
+    // Tests the --copyright call.
```

Comment:
Please put this into its own test case instead of adding it to the `TEST(help_add_test, add_option)` test.